app/models/discussion.rb

```diff
@@ -8,10 +8,6 @@ class Discussion < ActiveRecord::Base
 
   default_scope -> { order('updated_at DESC') }
 
-  after_save :update_subscriptions
```

Comment:
Sorry. I was reading in the email I mixed up the prs. It's ok with me. If we see this getting to slow, we can act on it later.